docs/guides/developer/docs/develop/debugging.md

```diff
@@ -1,12 +1,59 @@
-# In Construction
+# Debugging in Opencast
 
-Attaching a Remote Debugger to Karaf
-------------------------------------
+This guide will show you how to debug Opencast.
 
-To debug a running Opencast system, you can attach a remote debugger in your IDE (Eclipse or NetBeans, i.e.). For that
+
+## Debugging strategies
+
+There are different ways to debug Opencast. You can debug the backend and the frontend.
+The backend is the Java code while the frontend is the Angular code.
+
+### How to know what to debug?
+
+First, you want to know you to identify where is the problem. The frontend problems are mostly visible in the browser.
+These problems normally are JavaScript errors or problems with the user interface.
+The backend problems are mostly visible in the logs and the API calls. These problems are mostly exceptions or errors.
+
+The frontend and the backend are connected. The frontend calls the backend via REST endpoints.
+If the information from the API is not correct, is a problem of the backend.
+If the information is correct but, the frontend is not showing the information correctly, is a problem of the frontend.
+
+
+
+
+
+
+## Backend debugging
+
+### Logging configuration
+
+The logging configuration is in the `etc/org.ops4j.pax.logging.cfg` file.
+You can change the logging level of a package by adding the following line:
+
+    log4j.logger.org.opencastproject.capture=DEBUG
+
+Where `org.opencastproject.capture` is the package name and `DEBUG` is the logging level.
+
+The logging level can be one of the following:
+
+* `TRACE`
+* `DEBUG`
+* `ERROR`
+* `WARN`
+* `INFO`
+
+
+### Attaching a Remote Debugger
+
+There are different ways to debug the backend of Opencast.
+The easiest way is to attach a remote debugger here are some two ways to do that.
+
+#### For source installations
+
+To debug a running Opencast system, you can attach a remote debugger in your IDE (Eclipse, NetBeans, etc.). For that
 you have to enable the remote debugging in Karaf OSGI server that runs Opencast.
 
-You have to add "debug" as an additional paramenter to the Opencast start script:
+You have to add “debug” as an additional parameter to the Opencast start script:
```

Comment:
Smart quotes on debug should be reverted.

Reply:
Done, LanguageTool (The European Grammarly) always suggests me to change to smart quotes, is there any difference for Markdown?